### ISSUE TITLE
Anyone getting this error What to do to fix it

### ISSUE THREAD

**Issue by Broly619** [sc0ty/subsync#193]
![IMG_20240824_012629_432.png](https://github.com/user-attachments/assets/4aa29307-684f-4f7e-af35-0ede4cd6133c)

**Comment by eNdEmiOn** [comment]
Yes same issue here, I reported it a good long wile ago but it doesn't seem anyone is listening anymore. All I could do was move back to version 0.16.0 (https://github.com/sc0ty/subsync/releases) which does work for me.
